osu.Game/Users/UserGridPanel.cs

```diff
@@ -4,139 +4,233 @@
 #nullable disable
 
 using osu.Framework.Allocation;
+using osu.Framework.Extensions.Color4Extensions;
+using osu.Framework.Extensions.ObjectExtensions;
 using osu.Framework.Graphics;
+using osu.Framework.Graphics.Colour;
 using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.Shapes;
+using osu.Framework.Graphics.Sprites;
+using osu.Framework.Input.Events;
 using osu.Game.Online.API.Requests.Responses;
+using osu.Game.Overlays.Profile;
 using osu.Game.Overlays.Profile.Header.Components;
+using osu.Game.Rulesets;
 using osuTK;
+using osuTK.Graphics;
 
 namespace osu.Game.Users
 {
     public partial class UserGridPanel : ExtendedUserPanel
     {
-        private const int margin = 10;
+        private FillFlowContainer details;
+        private LevelBadge level;
+        private Box hoverDim;
+
+        [Resolved]
+        private RulesetStore rulesets { get; set; }
+
+        protected override bool ShouldCreateUserCoverBackground { get; set; } = false;
+
+        private const int main_panel_heigth = 60;
```

Comment:
typo in name, should be
```csharp
        private const int main_panel_height = 60;
```
not posting as suggestion as references also need updating